test/metabase/cmd_test.clj

```diff
@@ -13,51 +13,29 @@
     (is (fn? the-fxn))))
 
 (deftest import-test
-  (with-redefs [cmd/resolve-enterprise-command (constantly
-                                                (fn [& args]
-                                                  (cons 'f args)))]
+  (with-redefs [cmd/call-enterprise list]
     (testing "load (v1)"
       (testing "with no options"
-        (is (= '(f "/path/" {:mode :skip, :on-error :continue})
+        (is (= '(metabase-enterprise.serialization.cmd/v1-load "/path/" {:mode :skip, :on-error :continue})
                (cmd/load "/path/"))))
       (testing "with options"
-        (is (= '(f "/path/" {:num-cans :2})
-               (cmd/load "/path/" "--num-cans" "2")))
-        (testing "People can still set --v2 true"
```

Comment:
Maybe kind of an "executive decision" but I don't think this option is useful. It is also confusing, and not documented anywhere.

Reply:
More scars from the last-second split of command names vs. the `--v2` flag.